libraries/USBHost/USBHost/USBEndpoint.cpp

```diff
@@ -140,7 +140,7 @@ void USBEndpoint::queueTransfer()
 
     //Now add this free TD at this end of the queue
     state = USB_TYPE_PROCESSING;
-    td_current->nextTD = td_next;
+    td_current->nextTD = (hcTd*)td_next;
```

Comment:
What are these changes in this file and in others. I assume they are related to IAR (warnings fixes), but all in one commit , without any description.